3.0.0/README.md

```diff
@@ -88,37 +192,39 @@ An array of data files in GeoJSON format. {z}, {x} and {y}, if present, are repl
 }
 ```
 
-## 3.5 `description`
+## 3.8 `description`
 
 OPTIONAL. Default: null.
 
-A text description of the tileset. The description can contain any valid unicode character as described by the JSON specification [RFC 8259](https://tools.ietf.org/html/rfc8259).
+A text description of the set of tiles. The description can contain any valid unicode character as described by the JSON specification [RFC 8259](https://tools.ietf.org/html/rfc8259).
 
 ```JSON
 {
   "description": "We are volcanoes. When we women offer our experience as our truth, as human truth, all the maps change. There are new mountains."
```

Comment:
Realistic examples are much better for demonstrating usage. So for instance:

"description": "Highways, roads and vehicular tracks derived from OpenStreetMap."

Reply:
https://github.com/mapbox/tilejson-spec/pull/36/commits/5e75cf855792a11a4ecaf8d9ae3372dad03ac127 - let me know if you'd prefer me use a different example 😄